cfme/utils/appliance/implementations/ssui.py

```diff
@@ -1,10 +1,15 @@
+from inspect import isclass
```

Comment:
From #5866 (for my own sanity :) )

Reply:
do you want me to remove this?

Reply:
no no, it'll disappear when we merge 5866, or it'll mostly disappear (I submitted 2 commits to that PR, if you rebased before I did so you'll end up with a commit in this branch that looks like its un-doing my 2 commits)

Fine for now, just making a comment so me and other reviewers don't start to review this here